### PR TITLE
4.8.16 dev

### PR DESCRIPTION
**Pull request by ruksana2808**

### CODE REVIEW

File: src/main/java/com/igot/cb/designation/service/impl/DesignationServiceImpl.java [hunk 1]
```diff
@@ -436,13 +435,22 @@ public CustomResponse updateDesignation(JsonNode updateDesignationDetails) {
           response.setResponseCode(HttpStatus.OK);
           log.info("InterestServiceImpl::createInterest::persited interest in Pores");
           return response;
+        } else {
+          logger.error(Constants.NOT_FOUND);
```
Comment (Sahil-tarento) [inline src/main/java/com/igot/cb/designation/service/impl/DesignationServiceImpl.java]: The msg should be proper

File: src/main/java/com/igot/cb/designation/service/impl/DesignationServiceImpl.java [hunk 2]
```diff
@@ -436,13 +435,22 @@ public CustomResponse updateDesignation(JsonNode updateDesignationDetails) {
...
+          response.setMessage(Constants.NOT_FOUND);
+          response.setResponseCode(HttpStatus.BAD_REQUEST);
+          return response;
         }
+      } else {
+        logger.error(Constants.ID_NOT_FOUND);
```
Comment (Sahil-tarento) [inline src/main/java/com/igot/cb/designation/service/impl/DesignationServiceImpl.java]: The msg should proper

**Review by Sahil-tarento — CHANGES_REQUESTED** [review]

File: src/main/java/com/igot/cb/designation/service/impl/DesignationServiceImpl.java (hunk 1 above)
Comment (ruksana2808) [reply src/main/java/com/igot/cb/designation/service/impl/DesignationServiceImpl.java]: made changes

File: src/main/java/com/igot/cb/designation/service/impl/DesignationServiceImpl.java (hunk 2 above)
Comment (ruksana2808) [reply src/main/java/com/igot/cb/designation/service/impl/DesignationServiceImpl.java]: made changes

**Pull request merged**